.github/workflows/build.yml

```
@@ -77,7 +77,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ["1.21"]
+        go-version: ["1.21","1.22"]
```

```suggestion
        go-version: ["1.21","1.22"]
```

linux build on go 1.22 is working on the system not sure why `build-all` is failing. Maybe we are too early to upgrade. Will wait for week or so to see if other dependencies updates before debugging this.

---

fixed